package.json

```diff
@@ -11,13 +11,15 @@
     "type": "git",
     "url": "https://github.com/prebid/Prebid.js.git"
   },
+
```

Comment:
More unintended changes?

Reply:
There should be no changes to package.json or adapters.json in this PR.

Reply:
ok. Please remove them, then